cmd/controlplane/garbage_collector.go

```diff
@@ -31,10 +31,11 @@ func newGarbageCollectorCommand() *cobra.Command {
 			ctx := cmd.Context()
 
 			version := versionpkg.GetVersion()
-			log.WithFields(log.Fields{
-				"version": version.Version,
-				"commit":  version.GitCommit,
-			}).Info("Starting Kargo Garbage Collector")
+			log := logging.LoggerFromContext(ctx)
```

Comment:
See [this comment](https://github.com/akuity/kargo/pull/1791/files#r1556679011) -- same concern.